tests/integration/config/plugins/integration-tests/data/tests/tests_afterburn_flight.txt

```diff
@@ -121,6 +122,8 @@ test "Afterburner-flight - Simple depart land"
 			"test: steps to wait" > 0
 		navigate
 			"travel destination" Ruin
+		debug "Start landing"
```

Comment:
```suggestion
		debug "Started landing"
```

Reply:
The previous message is after the call to depart. In order to get there, we must have departed. This is before the call to land, and therefore past tense would be inappropriate.